SwiftCompilerSources/CMakeLists.txt

```diff
@@ -127,9 +127,10 @@ function(add_swift_compiler_modules_library name)
       ${SWIFT_HOST_VARIANT_ARCH} "${deployment_version}")
 
   # Let Swift discover SwiftShims headers which are included by some headers
-  # under `include/swift`.
+  # under `include/swift`. These are either located next to the compiler (in case of open source toolchains) or
+  # in the SDK (in case a Swift compiler from Xcode)
   get_filename_component(swift_exec_bin_dir ${ALS_SWIFT_EXEC} DIRECTORY)
-  set(sdk_option ${sdk_option} "-I" "${swift_exec_bin_dir}/../lib")
+  set(sdk_option ${sdk_option} "-I" "${swift_exec_bin_dir}/../lib" "-I" "${sdk_path}/usr/lib")
```

Comment:
Hmm, does this mean we'll include two copies of SwiftShims when using an open source toolchain with an SDK from Xcode? If that's true, can we instead add only one of these two directories if both exist?

Reply:
I have done that locally and it worked just fine. I assume that it just chooses the first include path that it finds.

Reply:
That makes sense, thanks.